src/sidebars/sidebar.html

```diff
@@ -17,8 +17,26 @@
             </div>
         </div>
     </div>
-    <div class="search {{sidebarCtrl.searchClass()}}" ng-controller="SearchCtrl as searchCtrl" ng-click="sidebarCtrl.searchClick()">
-        <div class="sidetab-top">
```

Comment:
The styling it was applying wasn't necessary and it didn't really make sense to leave it in considering the search can now appear at the bottom not at the top. So it made more sense to just remove the class than overriding the styles.